Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/MenuViewController.swift

```diff
@@ -198,7 +203,7 @@ extension MenuViewController: PanModalPresentable {
         return _scrollViewChild(in: topVC.view)
     }
     var longFormHeight: PanModalHeight {
-        .maxHeight
+        .maxHeightWithTopInset(32)
```

Comment:
Is this for this ticket or the last-bookmark-hidden one?

Reply:
That is for this ticket:

> Increases the top inset so there is always space to tap to dismiss the menu

It helps VoiceOver users or users with motor function disabilities to dismiss by tapping the background instead of being forced to dismiss by gesture